android/app/build.gradle

```diff
@@ -41,6 +42,7 @@ android {
         applicationId "network.adventuresin.meetup_chatapp"
         minSdkVersion 16
         targetSdkVersion 28
+        multiDexEnabled true
```

Comment:
I think we want the AndroidX version of adding multidex support, see:
https://developer.android.com/studio/build/multidex#mdex-pre-l 

If we bump the minSdkVersion to 21 the multiDex is enabled by default, see 
https://github.com/flutter/flutter/issues/21009#issuecomment-425295263

Personally that would be my preference (set minSdkVersion to 21).  It's really just a question of do we support KitKat: 
https://developer.android.com/about/dashboards

What do you think?

Reply:
It would make sense to bump the minSdkVersion to 21. It would simplify the configuration, and we really have no reason to support all the Android versions before Lollipop for this kind of project.
I'll look into it in an hour or so.